server/sonar-server/src/main/java/org/sonar/server/issue/index/IssueDoc.java

```diff
@@ -117,10 +117,17 @@ public Integer line() {
     return getNullableField(IssueIndexDefinition.FIELD_ISSUE_LINE);
   }
 
+  @Deprecated
   @Override
   @CheckForNull
   public Double effortToFix() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)